homeworks/vladyslav.makedon_Vladmaked/11-dependency-injection/vehicle-list/src/app/pipes/transform-vehicle-address.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {Vehicle} from '../models';
+
+@Pipe({
+  name: 'transformVehicleAddressPipe'
+})
+
+export class TransformVehicleAddressPipePipe implements PipeTransform {
+  transform(vehicle: Vehicle): string {
+    return `https://www.google.com/search?q=${vehicle.vehicle.replace(/ /g, '+')}`;
```

Comment:
It's better to use the `encodeURIComponent` function here 😄 
This function works with any complex URI